cmd/verifier/README.md

```diff
@@ -0,0 +1,30 @@
+# Verifying provenances
+
+The [`verifier`](/internal/verifier/) package provides functionality for verifying an input
+provenance file. The provenance file should follow the
+[Amber provenance](./../pkg/amber/schema/v1/provenance.json) format and provide a list of materials
```

Comment:
How about Amber? Maybe add a reference to #224?

Reply:
Done.